save-agent/src/commonMain/kotlin/com/saveourtool/save/agent/utils/FileUtils.kt

```diff
@@ -0,0 +1,74 @@
+/**
+ * Utility methods to work with files using Okio
+ */
+
+package com.saveourtool.save.agent.utils
```

Comment:
copy-paste?

Reply:
Not really understand, what's wrong?

Reply:
You have copy pasted these file Utils from somewhere? We not use common part or even from Native

Reply:
It's used in Request.kt in save-agent module